internal/cli/cmd/hubble.go

```diff
@@ -175,6 +175,7 @@ func newCmdUI() *cobra.Command {
 	}
 
 	cmd.Flags().IntVar(&params.UIPortForward, "port-forward", 12000, "Local port to use for the port forward")
+	cmd.Flags().BoolVar(&params.UIOpenBrowser, "open-browser", true, "When --open-browser is supplied as false, cilium hubble ui will not open the browser. Default is true which means cilium hubble ui will always open a browser to view the UI")
```

Comment:
nit: the help message will show as

```
--open-browser       When --open-browser is supplied as false, cilium hubble ui will not open the browser. Default is true which means cilium hubble ui will always open a browser to view the UI (default true)
```
The trailing `(default true)` part is already hinting to what happens when the flag isn't provided IMHO
```suggestion
	cmd.Flags().BoolVar(&params.UIOpenBrowser, "open-browser", true, "When --open-browser=false is supplied, cilium Hubble UI will not open the browser")
```